docs/transpile/index.mdx

```diff
@@ -0,0 +1,25 @@
+---
+title: Introduction to the transpiler
+description: Introduction to the transpiler
+
+---
+
+
+# Introduction to the transpiler
+
+Transpilation is the process of rewriting a given input circuit to match the topology of a specific quantum device and optimize the circuit instructions for execution on present-day noisy quantum systems.  
+
+A central component of Qiskit, the transpiler is designed for modularity and extensibility. The goal is to be able to easily write new circuit transformations (known as transpiler **passes**), and combine them with other existing passes.  Which passes are chained together and in which order has a major effect on the final outcome. This pipeline is determined by a **pass manager**, which schedules the passes and also allows passes to communicate with each other by providing a shared space. In this way, the transpiler opens up the door for research into aggressive optimization of quantum circuits.
```

Comment:
I am trying to think of a less idiomatic way to rephrase "opens up the door", but anything I think of is just as metaphoric 😆 (paves the way? Makes space for?.... not any better!) so I'll leave it for now unless you can think of something!

Reply:
Also changed this around to also speak to utility scale work.  Let me know what you all think!